code/modules/media/media_tracks.dm

```diff
@@ -0,0 +1,63 @@
+//
+// Load the list of available music tracks for the jukebox (or other things that use music)
+//
+
+// Music track available for playing in a media machine.
+/datum/track
```

Comment:
Is it possible for this to work with Baystation's [music track datum](https://github.com/discordia-space/CEV-Eris/tree/master/code/datums/music_tracks), and [licenses?](https://github.com/discordia-space/CEV-Eris/blob/master/code/datums/licences.dm)